src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -31,6 +31,8 @@ public sealed class PowerFxConfig
         {
             DebugName = "DefaultConfig"
         };
+        
+        internal IDictionary<TexlFunction, object /*AsyncFunctionPtr*/> ConfigDependentFunctions = new Dictionary<TexlFunction, object>();

```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

Nit, readonly?
Can we add a comment. This is mapping a shared TexlFunction to their implementation?

Should ParseJson support use this? (can do in a fuutre PR, but just thinking about direction).